pcsx2/Recording/InputRecordingControls.cpp

```diff
@@ -60,6 +60,7 @@ void InputRecordingControls::HandleFrameAdvanceAndPausing()
 	else if (frameCountTracker != g_FrameCount)
 	{
 		frameCountTracker = g_FrameCount;
+		g_InputRecording.LogAndRedraw();
```

Comment:
As we add more and more to this class, its becoming less about just being an interface for the key-bindings and more about handling frame-to-frame interactions.  In the future, we may want to split that up / pull it into `InputRecording.cpp`.  But this isn't something for this PR by any means.